learning_observer/prototypes/dash_wo_teacher_dashboard/wo_dash/pages/assignment_page/students.py

```diff
@@ -52,22 +52,33 @@ def student_dashboard_view(course_id, assignment_id):
                     # open settings button
                     html.Div(
                         [
-                            settings.open_btn
+                            dbc.Button(
```

Comment:
I would add these are items in the dropdown menu. That way the menu contains more than just logout. You may have to alter the `settings.open_btn` to address this. The settings open button also includes a cog (settings icon), that can probably be removed if its moved into the dropdown. There is a classname on the text of the open settings button that hides when the screen is small, this will need to be addressed as well.

Reply:
@JohnDamilola

Reply:
@bradley-erickson I'll be pushing it in a bit, but first, I want to advice that we shouldn't put the "last updated" status in the dropdown too for consistency sake since it is not a actionable/clickable button.

I have added the settings toggle button into the dropdown, if you give me a go ahead, I will just update my PR with that

Reply:
Sounds good

Reply:
I've updated it

Reply:
Can you wrap the button and the dropdown menu in a button group and include a picture with it as well?

See the dropdown section [here](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/button_group/).

Using outlines instead of full color buttons might look a little nicer as well

Reply:
I have wrapped it in the button group, but what do you mean by "include a picture with it as well"?

Reply:
Just take a screenshot. and include it here. That way we and whoever else looks at this PR can quickly see what's changed.

Reply:
UI Changes

![Screenshot 2022-12-23 at 13 34 33](https://user-images.githubusercontent.com/17693596/209392452-5fe31cf0-a09b-4525-af4d-8f1426d45a3f.png)
![Screenshot 2022-12-23 at 13 34 44](https://user-images.githubusercontent.com/17693596/209392454-7cd77383-7cc5-4677-810c-8caef7c8aae8.png)